src/engine/sync/synccontrol.cpp

```diff
@@ -14,6 +14,8 @@
 #include "util/logger.h"
 #include "util/math.h"
 
+const double kTrackPositionMasterHandoff = 0.99;
```

Comment:
What's the reason for this cutoff instead of using the actual end? Just wondering, because 0.01 can be very different of a 2 minute track compared to a premixed 2 hour set.

Reply:
@ywwg Please comment those decisions for others and your future self. Otherwise no one will able to reason why this particular value has been chosen.

Reply:
that's  a good point.  I think I was worried about a case where we hit 1.0 and the deck stops before the handoff occurs.  I think that's not an issue though